sqlmesh/magics.py

```diff
@@ -782,12 +782,22 @@ def create_test(self, context: Context, line: str) -> None:
         help="Only run tests that match the pattern of substring.",
     )
     @argument("--verbose", "-v", action="store_true", help="Verbose output.")
+    @argument(
+        "--persist-fixtures",
```

Comment:
"Persist" seems a little misleading. The fixtures are persisted regardless of whether the flag is provided. It's whether they are cleaned up is the question. Should this be `preserve` or something indicated the lack of cleanup.

For context, I initially thought that this command dumps the fixtures to some other place.

Reply:
Ah, I can see the confusion, thanks for letting me know. I think `--preserve-fixtures` looks good.